langchain/document_loaders/web_base.py

```diff
@@ -103,7 +120,16 @@ async def fetch_all(self, urls: List[str]) -> Any:
         for url in urls:
             task = asyncio.ensure_future(self._fetch_with_rate_limit(url, semaphore))
             tasks.append(task)
-        return await asyncio.gather(*tasks)
+        try:
+            from tqdm.asyncio import tqdm_asyncio
+
+            _runner = tqdm_asyncio
+        except ImportError:
+            warnings.warn("For better logging of progress, `pip install tqdm`")
+            _runner = asyncio
+        return await _runner.gather(
+            *tasks, desc="Fetching pages", ascii=True, mininterval=1
```

Comment:
Not sure if import fails and runner becomes `asyncio`, would it still support the custom arguments like `desc`, `ascii` and `mininterval` which are only tqdm specific?

Reply:
ah great point